packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1656,12 +1754,21 @@ abstract class GridRenderObjectWidgetBase extends MultiChildRenderObjectWidget {
   }
 }
 
-/// Uses the grid layout algorithm for its children.
+/// A widget that creates a grid using a custom [delegate].
 ///
 /// For details about the grid layout algorithm, see [RenderGrid].
-class CustomGrid extends GridRenderObjectWidgetBase {
-  CustomGrid({ Key key, List<Widget> children: _emptyWidgetList, this.delegate })
-    : super(key: key, children: children) {
+///
+/// To pass data to your delegates [GridDelegate.getChildPlacement] function,
+/// see [GridPlacementData].
```

Comment:
"to do x, see y" doesn't make sense. You don't just see y to do x. you see y to learn how to do x. :-)